src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -1082,14 +1082,14 @@ namespace nupic {
     setup(sp, 6, 6);
 
     Real initMinActiveDutyCycles1[] =
-      {1e-6, 1e-6, 1e-6, 1e-6, 1e-6};
+      {1e-6, 1e-6, 1e-6, 1e-6, 1e-6, 1e-6 };
     Real initActiveDutyCycles1[] =
       {0.1, 0.3, 0.02, 0.04, 0.7, 0.12};
     Real initBoostFactors1[] =
       {0, 0, 0, 0, 0};
     Real trueBoostFactors1[] =
       {1, 1, 1, 1, 1};
-    Real resultBoostFactors1[5];
+    Real resultBoostFactors1[6];
     sp.setMaxBoost(10);
```

Comment:
An additional change is required here to increase the other arrays to 6 entries. The SP is setup with 6 columns.